doc/api/fs.md

```diff
@@ -4467,6 +4467,33 @@ The `atime` and `mtime` arguments follow these rules:
 * If the value can not be converted to a number, or is `NaN`, `Infinity`, or
   `-Infinity`, an `Error` will be thrown.
 
+
+### `fs.walk(path[, options], callback)`
+
+<!-- YAML
+added: v20.4.0
+-->
+
+* `path` {string|Buffer|URL}
+* `options` {Object}
+  * `dir` {string\[]} Array of folder names to ignore when traversing the directory tree.
+  * `file` {string\[]} Array of file names to ignore when traversing the directory tree.
```

Comment:
Calling the directories and files to ignore `dir` and `file` seems very surprising. Any reason not to call it `ignore` or (if we really need to separate directories from files, which is also something I'm not so sure about) `ignoreDirs` and `ignoreFiles`?